### PR TITLE
Added "isMT" option to  SimConfig:

### PR DESCRIPTION
**Pull request by ihrivnac**
requires FairRoot dev with PR #727

### CODE REVIEW

**Comment by ihrivnac** [comment]
This PR requires FairRoot dev with PR #727 where the function FairRunSim::SetIsMT is added.

File: Common/SimConfig/include/SimConfig/SimConfig.h [hunk 1]
```diff
@@ -54,6 +54,7 @@ class SimConfig
   std::string getExtKinematicsFileName() const { return mExtKinFileName; }
   unsigned int getStartEvent() const { return mStartEvent; }
   float getBMax() const { return mBMax; }
+  double getIsMT() const { return mIsMT; }
```
Comment (sawenzel) [inline Common/SimConfig/include/SimConfig/SimConfig.h]: shouldn't this return a `bool`?

File: Detectors/gconfig/g4Config.C [hunk 2]
```diff
@@ -39,7 +39,7 @@ void Config()
 /// character: eg. stepLimit+specialCuts.
 
    //Geant4 VMC 3.x
-   Bool_t mtMode = false;
+   Bool_t mtMode = FairRunSim::Instance()->IsMT();
```
Comment (sawenzel) [inline Detectors/gconfig/g4Config.C]: why not just accessing `confref` here?

Comment (ihrivnac) [reply Detectors/gconfig/g4Config.C]: As fairRunSim class holds alredy the selection of MC, so it seemed to me the best plece where to put this one

**Comment by sawenzel** [comment]
@dberzano : Can you please take care of the FairRoot bump necessary for this PR?

**Pull request merged**